app/scripts/modules/core/src/application/search/ApplicationsTable.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import { UISref } from '@uirouter/react';
+
+import { timestamp } from 'core/utils';
+import { IApplicationSummary } from 'core/application';
+
+import { ColHeader } from './ColHeader';
+
+export interface IApplicationTableProps {
+  applications: IApplicationSummary[];
+  currentSort: string;
+  toggleSort: (column: string) => void;
+}
+
+export const ApplicationTable = ({ currentSort, toggleSort, applications }: IApplicationTableProps) => (
+  <table className="table table-hover">
+    <thead>
+    <tr>
+      <ColHeader toggleSort={toggleSort} currentSort={currentSort} width="18%" label="Name" column="name"/>
+      <ColHeader toggleSort={toggleSort} currentSort={currentSort} width="15%" label="Created" column="createTs"/>
+      <ColHeader toggleSort={toggleSort} currentSort={currentSort} width="15%" label="Updated" column="updateTs"/>
+      <ColHeader toggleSort={toggleSort} currentSort={currentSort} width="15%" label="Owner" column="email"/>
+      <ColHeader toggleSort={toggleSort} currentSort={currentSort} label="Account(s)" column="accounts"/>
```

Comment:
I created a `SortToggle` component for React - we should either reuse that or use this where we are using the one I created (I don't have an opinion on which is better)